bundles/org.openhab.binding.tado/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -301,20 +356,32 @@
 		<label>Overlay End Time</label>
 		<description>Time until when the overlay is active. Null if no overlay is set or overlay type is manual.</description>
 		<category>Time</category>
+		<tags>
+			<tag>Control</tag>
```

Comment:
The chanenl is read-ony, so I guess not `Control`?

Reply:
Good catch! Fixed